react-frontend/src/components/Charts.tsx

```diff
@@ -336,17 +336,21 @@ function Charts({ a_n, b_n, limit, symbol, convergesTo, deltaData, toggleDisplay
 				) : (
 					''
 				)}
-				<div className="top-padding plot-container">
-					<p>
-						Delta is a measure of the irrationality of a number (read more about it{' '}
-						<a href="https://www.ramanujanmachine.com/the-mathematics-of-polynomial-continued-fractions/irrationality-testing/">
-							here
-						</a>
-						). The given Polynomial Continued Fraction produces the following finite-depth
-						estimations for Delta:
-					</p>
-					<ScatterPlot id="delta_chart" data={deltaData} />
-				</div>
+				{deltaData && deltaData?.length > 0 ? (
```

Comment:
not showing the chart if the data fails to load on the backend (they have a bug that sometimes crashes divide by zero)